<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalendarAttrCfger</class>
 <widget class="QGroupBox" name="CalendarAttrCfger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>230</height>
   </rect>
  </property>
  <property name="title">
   <string>Calendar</string>
  </property>
  <property name="alignment">
   <set>Qt::AlignCenter</set>
  </property>
  <property name="flat">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Jet Lag:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="offset">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>190</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string notr="true">+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">-</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="space">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Space:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="dateFirst">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Date first</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="date">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dateFormats">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="ColorComboBox" name="dateColor">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="ColorComboBox" name="weekColor">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="week">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Week</string>
   </property>
  </widget>
  <widget class="QComboBox" name="weekFormats">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="ColorComboBox" name="timeColor">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="timeFormats">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QPushButton" name="underLine">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Underline</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/icon/text_underline_64.png</normaloff>image/icon/text_underline_64.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bold">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Bold</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/icon/text_bold_64.png</normaloff>image/icon/text_bold_64.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="point">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Font size</string>
   </property>
   <property name="maximum">
    <number>512</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QFontComboBox" name="family">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="fontFilters">
    <set>QFontComboBox::MonospacedFonts</set>
   </property>
  </widget>
  <widget class="QPushButton" name="italic">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Italic</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>image/icon/text_italic_64.png</normaloff>image/icon/text_italic_64.png</iconset>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="ColorComboBox" name="textColor">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="extraText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Extra text</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>241</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="textColorTag">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Color:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="multiLine">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Multiple line</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="offsetValue">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>190</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string notr="true">H:mm</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorComboBox</class>
   <extends>QComboBox</extends>
   <header>ColorComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
